pages/guideforms.py

```diff
@@ -66,6 +66,21 @@ def validate(self, value):
         for url in value:
             validate_url(url.strip())
 
+class ChromedashTextInput(forms.widgets.Input):
+    template_name = 'django/forms/widgets/text.html'
+
+class ChromedashTextarea(forms.widgets.Textarea):
+    template_name = 'django/forms/widgets/chromedash-textarea.html'
+
+    def __init__(self, attrs=None):
+        # Use slightly better defaults than HTML's 20x2 box
+        default_attrs = {
+            'cols': 50, 'rows': 10,  'maxlength': 500,
+            'placeholder': SUMMARY_PLACEHOLDER_TXT}
```

Comment:
It seems oddly specific to have the class definition default to using a placeholder that is meant for the summary field.   How about having no placeholder specified in the class but add back `widget=ChromedashTextarea(attrs={'placeholder': SUMMARY_PLACEHOLDER_TXT})` on line 159.

Reply:
`SUMMARY_PLACEHOLDER_TXT` has instructions specific to the summary field. We definitely DON'T want that placeholder text in other fields.